public/app/features/canvas/runtime/SceneTransformWrapper.tsx

```diff
@@ -13,10 +13,37 @@ type SceneTransformWrapperProps = {
 export const SceneTransformWrapper = ({ scene, children: sceneDiv }: SceneTransformWrapperProps) => {
   const onZoom = (zoomPanPinchRef: ReactZoomPanPinchRef) => {
     const scale = zoomPanPinchRef.state.scale;
+    scene.scale = scale;
+  };
+
+  const onZoomStop = (zoomPanPinchRef: ReactZoomPanPinchRef) => {
+    const scale = zoomPanPinchRef.state.scale;
+    scene.scale = scale;
+    updateMoveable(scale);
+  };
```

Comment:
@drew08t i've done some minor cleanup to these changes, would appreciate a quick set of 👀 to make sure I'm not doing anything strange 😬